tools/genomespace/genomespace_importer.py

```diff
@@ -192,7 +191,7 @@ def download_from_genomespace_importer( username, token, json_parameter_file, ge
         # if using tmp file, move the file to the new file path dir to get scooped up later
         if using_temp_file:
             original_filename = filename
-            filename = ''.join( c in VALID_CHARS and c or '-' for c in filename )
+            filename = ''.join( c in FILENAME_VALID_CHARS and c or '-' for c in filename )
```

Comment:
Sounds good.  I'm just going to revert the genomespace changes for now since I really want to hear from @blankenberg on that and I don't want to hold this PR up any more.

Will follow up on extending the valid character set in a separate endeavor.